libs/Microsoft.MixedReality.WebRTC/Interop/InteropUtils.cs

```diff
@@ -291,5 +291,25 @@ public static string EncodeTransceiverStreamIDs(List<string> streamIDs)
             }
             return string.Join(";", streamIDs.ToArray());
         }
+
+        [StructLayout(LayoutKind.Sequential)]
+        internal struct H264Config
+        {
+            internal int Profile;
+            internal int RcMode;
+            internal int MaxQp;
+            internal int Quality;
+
+            internal H264Config(PeerConnection.H264Config config)
+            {
+                Profile = (int)config.Profile;
+                RcMode = config.RcMode.HasValue ? (int)config.RcMode : -1;
+                MaxQp = config.MaxQp.HasValue ? (int)config.MaxQp : -1;
+                Quality = config.Quality.HasValue ? (int)config.Quality : -1;
```

Comment:
nit: Use `GetValueOrDefault(-1)` instead for clarity?

Reply:
Fixed